examples/demo_vlan.pp

```diff
@@ -16,9 +16,10 @@
 
 class ciscopuppet::demo_vlan {
   cisco_vlan { '220':
-    ensure    => present,
-    vlan_name => 'newtest',
-    shutdown  => true,
-    state     => 'active',
+    ensure     => present,
```

Comment:
Please update this demo_manifest to use the platform_get() api to only set the `mapped_vni` prop if the platform supports it.

Example:

https://github.com/cisco/cisco-network-puppet-module/blob/develop/examples/demo_portchannel.pp#L44